src/entities/__tests__/mellowLpRouter.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { providers, Wallet } from "ethers";
```

Comment:
great stuff! But I have one question... what is the purpose of adding the tests if the pipeline doesn't run them? How do we ensure that tests are not gonna get broken in the next PR?

Reply:
That is a good point. So far I have added rudimentary tests which should check the setup and basic functionality. These should be running fine once the contracts are deployed. But if these are being run in the github ci pipeline before that time, then they will fail. Wdyt @0xZenus ?

Reply:
I think it is fine for them to fail if the "infrastructure" is not there. And we should not merge your PR until they pass :D Let's bring more quality code into main.

Can I ask you and @0xZenus to add the pipeline check for tests?

Reply:
I think we can add the tests to the pipeline. However, the tests should pass once the deployments are going to be live. This PR is preliminary at this moment.

Reply:
I changed the yarn test command to reflect npx hardhat test